transport/src/main/java/io/netty/channel/AbstractChannel.java

```diff
@@ -405,6 +406,11 @@ public final ChannelPromise voidPromise() {
         return pipeline.voidPromise();
     }
 
+    final void autoFlushModified(boolean autoFlush) {
```

Comment:
that said I think its ok... if people change it concurrently there is not much we can do I guess.